File: src/test/java/com/example/travelator/travel/java/MarketingTest.java
package com.example.travelator.travel.java;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class MarketingTest {
	
	@DisplayName("should return true when domain is hotmail.com")
	@Test
	void testCase1() {
		EmailAddress mail = new EmailAddress("seok", "hotmail.com");
		
		boolean actual = Marketing.isHotMailAddress(mail);
		
		assertThat(actual).isTrue();
	}
}

File: src/test/java/com/example/travelator/travel/kotlin/MarketingTest.kt
package com.example.travelator.travel.kotlin

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

internal class MarketingTest {

    @DisplayName("should recognize hotmail addresses")
    @Test
    internal fun testCase1() {
        val emailAddress = EmailAddress.parse("seok@hotmail.com")
        val actual = Marketing.isHotMailAddress(emailAddress)
        assertThat(actual).isTrue
    }

}